sql/internal/specutil/types.go

```diff
@@ -80,3 +84,81 @@ func (r *TypeRegistry) Find(t string) (*schemaspec.TypeSpec, bool) {
 	}
 	return nil, false
 }
+
+// Convert converts the schema.Type to a *schemaspec.Type.
+func (r *TypeRegistry) Convert(typ schema.Type) (*schemaspec.Type, error) {
+	s := &schemaspec.Type{}
+	vof := reflect.ValueOf(typ)
```

Comment:
```suggestion
	rv := reflect.ValueOf(typ)
```

rv (reflect value) is more common.